apps/backoffice-v2/src/pages/Entities/Entities.page.tsx

```diff
@@ -40,7 +40,7 @@ export const Entities: FunctionComponent = () => {
               ? skeletonEntities.map(index => (
                   <Cases.SkeletonItem key={`cases-list-skeleton-${index}`} />
                 ))
-              : cases?.map(case_ => (
+              : (Array.isArray(cases) ? cases : []).map(case_ => (
```

Comment:
Why this change? React doesn't render undefined and optional chaining prevents map from failing.

Reply:
I don't know why, but when developing I had ESLint issues there, but now I don't. Weird.